yarn.lock

```diff
@@ -0,0 +1,3471 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Why yarn?

Reply:
Generally I am more accustomed to using yarn over npm, it is significantly faster in my experience, but if you'd like I can use npm for every Kaholo repo.

Reply:
I prefer `Yarn` also, but if we decide to introduce such change, then this should be done for all the plugins. Let's keep `npm` for now.